extension/manifest.json

```diff
@@ -75,7 +75,7 @@
       "js": ["shared.js","keysocket-grooveshark.js"]
     },
     {
-      "matches": ["https://www.amazon.com/gp/dmusic/mp3/player*"],
+      "matches": ["https://www.amazon.com/gp/dmusic/cloudplayer/web*"],
```

Comment:
I don't think the change has gone public, yet. The chrome extension page still has a March 25th date.

https://chrome.google.com/webstore/detail/key-socket-media-keys/fphfgdknbpakeedbaenojjdcdoajihik